lib/active_remote/base.rb

```diff
@@ -23,22 +26,19 @@ module ActiveRemote
   class Base
     extend ActiveModel::Callbacks
 
-    include ActiveAttr::BasicModel
-    include ActiveAttr::Attributes
-    include ActiveAttr::BlockInitialization
-    include ActiveAttr::ChainableInitialization
-    include ActiveAttr::Logger
-    include ActiveAttr::MassAssignment
-    include ActiveAttr::AttributeDefaults
-    include ActiveAttr::QueryAttributes
-    include ActiveAttr::Serialization
+    extend ::ActiveModel::Naming
+    include ActiveModel::Conversion
```

Comment:
I was planning on fixing this to be explicit for the entire project later...I can do that now though?